user_manual/docs/rdm-responder.md

```diff
@@ -92,7 +92,7 @@ Sensor Index | Name           | Type, Units                     | Units   | Note
 -------------|----------------|---------------------------------|---------|---------
 0            | Temperature    | Temperature, Centigrade         | Deci    | Data is from the onboard temp. sensor if present.
```

Comment:
Should we add a note about this being random data if the sensor isn't fitted (although I still really think it should NAck in that case.